### PR TITLE
Update animation codelab

### PR DESCRIPTION
**Pull request by riggaroo**
* Use Material 3
* Change colors to be a bit softer
* Enable Edge-to-Edge

### CODE REVIEW

File: AnimationCodelab/finished/src/main/java/com/example/android/codelab/animation/ui/home/Home.kt [hunk 1]
```diff
@@ -178,7 +184,7 @@ fun Home() {
     val lazyListState = rememberLazyListState()
 
     // The background color. The value is changed by the current tab.
-    val backgroundColor by animateColorAsState(if (tabPage == TabPage.Home) Purple100 else Green300)
+    val backgroundColor by animateColorAsState(if (tabPage == TabPage.Home) Seashell else GreenLight)
```
[the remark this answers is not in the document]
Comment (riggaroo) [reply AnimationCodelab/finished/src/main/java/com/example/android/codelab/animation/ui/home/Home.kt]: done

File: AnimationCodelab/finished/src/main/java/com/example/android/codelab/animation/ui/home/Home.kt [hunk 2]
```diff
@@ -202,69 +208,75 @@ fun Home() {
...
-            items(count = tasks.size) { i ->
-                val task = tasks.getOrNull(i)
-                if (task != null) {
-                    key(task) {
-                        TaskRow(
-                            task = task,
-                            onRemove = { tasks.remove(task) }
-                        )
+                items(count = tasks.size) { i ->
+                    val task = tasks.getOrNull(i)
```
[the remark this answers is not in the document]
Comment (riggaroo) [reply AnimationCodelab/finished/src/main/java/com/example/android/codelab/animation/ui/home/Home.kt]: done

File: AnimationCodelab/start/src/main/java/com/example/android/codelab/animation/ui/home/Home.kt [hunk 3]
```diff
@@ -182,69 +188,75 @@ fun Home() {
...
-            }
-            items(count = tasks.size) { i ->
-                val task = tasks.getOrNull(i)
-                if (task != null) {
-                    key(task) {
-                        TaskRow(
-                            task = task,
-                            onRemove = { tasks.remove(task) }
-                        )
+                items(count = tasks.size) { i ->
```
[the remark this answers is not in the document]
Comment (riggaroo) [reply AnimationCodelab/start/src/main/java/com/example/android/codelab/animation/ui/home/Home.kt]: done

File: AnimationCodelab/finished/src/main/java/com/example/android/codelab/animation/ui/home/Home.kt [hunk 4]
```diff
@@ -202,69 +211,70 @@ fun Home() {
...
-                    }
-                )
-            }
-            item { Spacer(modifier = Modifier.height(32.dp)) }
-
-            // Tasks
-            item { Header(title = stringResource(R.string.tasks)) }
-            item { Spacer(modifier = Modifier.height(16.dp)) }
-            if (tasks.isEmpty()) {
+        Box(Modifier
```
Comment (JolandaVerhoef) [inline AnimationCodelab/finished/src/main/java/com/example/android/codelab/animation/ui/home/Home.kt:214]: This doesn't add edge-to-edge completely (the bottom is still cut off). Instead, you could consider something like:
```
Box(Modifier.padding(top = padding.calculateTopPadding())) {
            LazyColumn(
                contentPadding = WindowInsets(
                    padding.calculateLeftPadding(LayoutDirection.Ltr) + 16.dp,
                    32.dp,
                    padding.calculateRightPadding(LayoutDirection.Ltr) + 16.dp,
                    padding.calculateBottomPadding() + 32.dp
                ).asPaddingValues(),
                state = lazyListState
            ) {
```

**Pull request merged**